netflix/src/index.css

```diff
@@ -14,8 +15,55 @@ code {
 
 h1 {
   text-align: center;
+  color: #191414;
 }
 
 h2 {
   text-align: center;
+  color: #191414;
 }
+
+.counter {
+  text-align: center;
+  display: grid;
+  grid-template-columns: 5rem 10rem 5rem;
+}
+
+button {
+  background-color: #1db954;
+  border-radius: 5rem;
+  transition: background-color 1s;
+}
+
+button:hover {
+  background-color: #ffffff;
+  border-radius: 2rem;
+}
+
+.navbar {
```

Comment:
Not a bug.

Usually, you want to keep your component styling as close to your component as possible. I see that in this case you made styles very global (i.e. it will apply to all buttons/labels), but for real-world projects, consider separating styles per components.